lib/tasks/curator.rake

```diff
@@ -0,0 +1,21 @@
+namespace :curator do
+  desc <<-DESC.strip_heredoc
+    Sets the curator for a given scope, curated highlights are available by querying the set
+    "curated:openstax".
+
+    eg: rake curator:set[scope_uuid,curator_uuid]
+  DESC
+  task :set, [:scope_id, :curator_id] => :environment do |task, args|
+    CuratorScope.where(:scope_id => args[:scope_id]).delete_all
```

Comment:
1. More modern to use more compact syntax: `where(scope_id: args[:scope_id])`
2. Use `destroy_all` instead of `delete_all`.  The `destroy` methods call callbacks on the models (e.g. there could be a `before_destroy` callback that checks that it is ok to delete the model).  The `delete` methods don't hit these callbacks.